packages/eslint-plugin/configs/recommended-with-formatting.js

```diff
@@ -45,19 +40,4 @@ const config = {
 	},
 };
 
-if ( isPackageInstalled( 'jest' ) ) {
```

Comment:
Unit tests are out of scope so it looks like the default handling was removed by accident. Let's bring it back.

Reply:
As mentioned in https://github.com/WordPress/gutenberg/pull/38570#discussion_r834133216, I think it's problematic in the long term to rely on environment sniffing. We can deprecate it in another PR though.

Reply:
I fully disagree here. The goal always was to make the configuration work when someone installs `@wordpress/scripts` so they don't have to think about what else they need to include. That was a collective decision to use this approach to lower the entry barrier. We use it also in other places and we shouldn't change that only because you have a different view on that topic.

Reply:
I misinterpreted your [message](https://github.com/WordPress/gutenberg/pull/38570#discussion_r827032819) here and thought that it's recommended to remove the custom handling of different systems. I didn't change it simply because I have a different view on this, it's just that I misunderstood it.